package.json

```diff
@@ -13,10 +13,11 @@
   "dependencies": {
     "moment": "^2.17.1",
     "moment-timezone": "^0.5.11",
-    "react": "15.4.2",
-    "react-native": "^0.42.0",
+    "react": "16.0.0-alpha.6",
+    "react-native": "^0.44.0",
     "react-native-blur": "^2.0.0",
-    "react-native-code-push": "^1.17.0-beta",
+    "react-native-code-push": "^2.0.3-beta",
```

Comment:
I had some issue with building on `1.17.0-beta` Sure, I'll try downgrading and see if I it still happens.